app/services/summary_service.rb

```diff
@@ -75,10 +75,26 @@ def self.summary(summary_type, prison, page, params)
       summary.page_count = page_count
     }
   end
-# rubocop:enable Metrics/CyclomaticComplexity
-# rubocop:enable Metrics/LineLength
-# rubocop:enable Metrics/MethodLength
-# rubocop:enable Metrics/PerceivedComplexity
+  # rubocop:enable Metrics/CyclomaticComplexity
+  # rubocop:enable Metrics/LineLength
+  # rubocop:enable Metrics/MethodLength
+  # rubocop:enable Metrics/PerceivedComplexity
+
+  def self.entered_prison_dates(offender_list)
+    return if offender_list.empty?
+
+    offenders = offender_list.map(&:offender_no)
+
+    return if offenders.nil?
```

Comment:
This can't actually happen - map cannot return nil

Reply:
Fixed in the latest commit